src/shared/extensionGlobals.ts

```diff
@@ -26,6 +27,7 @@ export namespace ext {
     export let telemetry: TelemetryService
     export let templateRegistry: CloudFormationTemplateRegistry
     export let codelensRootRegistry: CodelensRootRegistry
+    export let fileViewerManager: S3FileViewerManager
```

Comment:
same comment as https://github.com/aws/aws-toolkit-vscode/pull/1855#discussion_r663270454 
```suggestion
    export let s3FileViewerManager: S3FileViewerManager
```